src/core.jl

```diff
@@ -104,4 +104,27 @@ function match_keypoints(keypoints_1::Keypoints, keypoints_2::Keypoints, desc_1,
         end
     end
     matches
+end
+
+"""
+```
+grade = grade_matches(keypoints_1, keypoints_2, difference_method)
+```
+
+Returns a measure of similarity between keypoints in `keypoints_1` and `keypoints_2` where `difference_method`
+is the method used for computing difference between individual pair of keypoints.
+"""
+
+function grade_matches(K1::Keypoints, K2::Keypoints, diff::Function = (i,j)->(abs(i[1] - j[1]) + (abs(i[2] - j[2]))))
```

Comment:
@mronian Even though lengths of K1 and K2 are equal they might be at very different angles, so shouldn't we check for closeness between keypoints rather than only comparing their length. The [previous tests](https://github.com/JuliaImages/ImageFeatures.jl/blob/master/test/orb.jl#L36) also compared for closeness by comparing both `rk[1]` to `m[2][1]` and `rk[2]` to `m[2][2]` with the `isapprox` function.